contrib/gorilla/mux/mux_test.go

```diff
@@ -11,70 +13,67 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestHttpTracer200(t *testing.T) {
-	assert := assert.New(t)
-	mt := mocktracer.Start()
-	defer mt.Stop()
+func TestHttpTracer(t *testing.T) {
+	for _, ht := range []struct {
+		code         int
+		method       string
+		url          string
+		resourceName string
+		errorStr     string
+	}{
+		{http.StatusOK, "GET", "/200", "GET /200", ""},
```

Comment:
Please remove the `""` at the end, we don't need it. The default value of a string is an empty string.